### PR TITLE
Translations: Fix strings for the repeats plugin

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: translations/english/repeats.ts [hunk 1]
```diff
@@ -3,18 +3,22 @@ import type {Translations} from '../../server/chat';
...
-		"${user.name} set the phrase \"${message}\" to be repeated every ${interval} minute(s).": "",
-		"The phrase \"${target}\" is not being repeated in this room.": "",
-		"${user.name} removed the repeated phrase \"${target}\".": "",
+		"You must specify an interval as a number of minutes between 1 and 1440.": "",
+		"The phrase labeled with \"${id}\" is already being repeated in this room.": "",
+		"${user.name} set the phrase labeled with \"${id}\" to be repeated every ${interval} minute(s).": "",
+		"${user.name} set the Room FAQ \"${topic}\" to be repeated every ${interval} minute(s).": "",
+		"The phrase labeled with \"${id}\" is not being repeated in this room.": "",
+		"The text for the Room FAQ \"${topic}\" is already being repeated.": "",
+		"${user.name} removed the repeated phrase labeled with \"${id}\".": "",
```
Comment (AnnikaCodes) [inline translations/english/repeats.ts:21]:
```suggestion
		"The phrase labeled with \"${id}\" is already being repeated in this room.": "",
		"${user.name} set the phrase labeled with \"${id}\" to be repeated every ${interval} minute(s).": "",
		'${user.name} set the Room FAQ "${topic}" to be repeated every ${interval} minute(s).': "",
		'The phrase labeled with "${id}" is not being repeated in this room.': "",
		'The text for the Room FAQ "${topic}" is already being repeated.': "",
		'${user.name} removed the repeated phrase labeled with "${id}".': "",
```

Comment (AnnikaCodes) [reply translations/english/repeats.ts:21]: This is TypeScript now, not JSON :P

Comment (KrisXV) [reply translations/english/repeats.ts:21]: fair, i was just being consistent with all of the other files

Comment (AnnikaCodes) [reply translations/english/repeats.ts]: which other files? I should make a commit to fix them

Comment (KrisXV) [reply translations/english/repeats.ts]: all

**Comment by Zarel** [comment]
Thanks!

I considered leaving this open for Annika to review, but honestly if there's a problem with this I'm sure she can just commit a fix directly.

**Pull request merged**